### ISSUE TITLE
date picker year/month mode width issue in IE

### ISSUE THREAD

**Issue by dguthrie10** [mgcrea/angular-strap#1506]
When using the datepicker in IE11, switching from mode 0 (month/day view) to mode 1 (year/month view), the width of the content is compressed. If you close the datepicker with mode 1 active, then reopen it, the content sizes to the full width of the container. This only happens in IE. Here is a plunkr to demonstrate:
http://plnkr.co/edit/0TzI0E?p=preview
![2-13-2015 10-03-17 am](https://cloud.githubusercontent.com/assets/5486407/6189392/2fcd83dc-b368-11e4-8a9d-b3d9a534ea76.png)
![2-13-2015 10-10-03 am](https://cloud.githubusercontent.com/assets/5486407/6189453/b3deabec-b368-11e4-9e8c-bbe6d5c1abe8.png)

**Comment by RobJacobs** [comment]
I've been seeing this issue to, IE11. If you navigate to another mode and go back, it's sized correctly. Even just using developer tools to modify a style attribute on the parent (text-align) will cause it to redraw correctly.

**Comment by bstruthers** [comment]
What's very odd about this issue is I can't get it to reproduce in plnkr when accessing the content directly.  http://run.plnkr.co/plunks/0TzI0E/

When its inside an iframe I can see the issue.

**Comment by bstruthers** [comment]
Looks like the issue is related to using `table-layout: fixed;` on the table in `datepicker.tpl.html`.  See http://plnkr.co/edit/9WiyNK92Dk1b983EJyON?p=preview

**Comment by dguthrie10** [comment]
Yes - you are right. It indeed is the table layout in the template. When you remove the fixed term, the sizing issue goes away, although it causes some other downstream effects. But - the issue is a styling one - not a functional one inside the component itself. I would call this one solved!

**Comment by bstruthers** [comment]
@dguthrie10 What downstream effects did you see?  When I looked at it with IE and Firefox, I didn't notice anything.

**Comment by dguthrie10** [comment]
The biggest thing I saw was the fwd/bkwrd buttons in the header of the template styled with class btn caused the forward/backward buttons to be too wide and resulted in the first column and last column of the table body being wider than the others. I removed the class=btn callouts and the situation resolved. Can't remember anything else specific. I played around quite a bit with the template to get just the look I wanted, so I might have caused some of the effects myself - but, knowing it was all just styling issues, none of it was difficult.
